.travis.yml

```diff
@@ -1,16 +1,11 @@
 language: ruby
 sudo: false
 rvm:
-  - 2.3.4
   - 2.4.1
 gemfile:
-  - gemfiles/rails_4.2.gemfile
```

Comment:
Lit needs to preserve Rails 4.2 compatibility, so this needs to be tested as well